.github/workflows/ci-actions-incremental.yml

```diff
@@ -160,7 +160,7 @@ jobs:
         # mvnw just for creating gib-impacted.log ("validate" should not waste much time if not incremental at all, e.g. on main)
         run: |
           ./mvnw -q -T1C $COMMON_MAVEN_ARGS -Dtcks -Dquickly-ci ${{ steps.get-gib-args.outputs.gib_args }} -Dgib.logImpactedTo=gib-impacted.log validate
-          [ -f gib-impacted.log ] && GIB_IMPACTED=$(sed "s|$(pwd)/||" gib-impacted.log) || GIB_IMPACTED=''
+          [ -f gib-impacted.log ] && GIB_IMPACTED=$(cat gib-impacted.log) || GIB_IMPACTED=''
```

Comment:
Removal of repo path prefix is not required anymore due to https://github.com/gitflow-incremental-builder/gitflow-incremental-builder/issues/353